spec.html

```diff
@@ -8504,6 +8524,7 @@ <h1>OrdinaryFunctionCreate ( _functionPrototype_, _ParameterList_, _Body_, _this
         1. Set _F_.[[Environment]] to _Scope_.
         1. Set _F_.[[ScriptOrModule]] to GetActiveScriptOrModule().
         1. Set _F_.[[Realm]] to the current Realm Record.
+        1. Set _F_.[[HomeObject]] to *undefined*.
```

Comment:
This seems unneeded in light of the following sentence in [6.1.7.2](https://tc39.es/ecma262/#sec-object-internal-methods-and-internal-slots):

"Unless specified otherwise, the initial value of an internal slot is the value undefined. "